sos/plugins/pci.py

```diff
@@ -23,6 +24,9 @@ def setup(self):
             "/proc/bus/pci"
         ])
 
+        self.set_cmd_predicate(SoSPredicate(self, arch=['ppc*', 's390*'],
```

Comment:
I'm not certain that this is the right way of handling this - the PPC platform has used PCI as the expansion bus for a long time now, and even Zseries systems can use PCI expansion devices in certain configurations. I believe this is the reason that we build the `pciutils` package for these arches.

Perhaps we should instead gate on the presence of bus directories in `/proc/bus/pci`, or on the output of `lspci` itself? I'm concerned that if we turn all these commands off uniformly for these platforms then we will be losing valuable debugging data for some valid configurations.